os/binfmt/elf.c

```diff
@@ -158,7 +158,7 @@ static void elf_dumploadinfo(FAR struct elf_loadinfo_s *loadinfo)
 
 	if (loadinfo->shdr && loadinfo->ehdr.e_shnum > 0) {
 		for (i = 0; i < loadinfo->ehdr.e_shnum; i++) {
-			FAR Elf32_Shdr *shdr = &loadinfo->shdr[i];
+			__attribute__((unused)) FAR Elf32_Shdr *shdr = &loadinfo->shdr[i];
```

Comment:
Can you pls remove all these unused attribute definitions? and make a proper fix for the warning?

Reply:
ok